coderd/users_test.go

```diff
@@ -94,7 +100,8 @@ func TestFirstUser(t *testing.T) {
 		}
 		_, err := client.CreateFirstUser(ctx, req)
 		require.NoError(t, err)
-		<-called
+		<-trialGenerated
+		<-entitlementsRefreshed
```

Comment:
suggest using `testutil.RequireRecvCtx` to avoid the test hanging indefinitely.

Reply:
Fixed 👍